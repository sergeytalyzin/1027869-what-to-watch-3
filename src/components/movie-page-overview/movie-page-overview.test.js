import React from "react";
import renderer from "react-test-renderer";
import MoviePageOverview from "./movie-page-overview.jsx";

const film = {
  id: 1,
  title: `Mindhunter`,
  src: `somePath`,
  genre: `Comedy`,
  date: 1812,
  posterBig: `img/mindhunter.jpg`,
  rating: 3,
  ratingLevel: `Very good`,
  ratingCount: 111,
  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Cras aliquet varius magna, non porta ligula feugiat eget.
  Fusce tristique felis at fermentum pharetra.
  Aliquam id orci ut lectus varius viverra.
  Nullam nunc ex, convallis sed finibus eget, sollicitudin eget ante.
  Phasellus eros mauris, condimentum sed nibh vitae, sodales efficitur ipsum.
  Sed blandit, eros vel aliquam faucibus, purus ex euismod diam, eu luctus nunc ante ut dui.`,
  actors: `Frank Sinatra, John Mason, Anthony Mann, Bred Pitt, Heinz Herald`,
  director: `Frank Sinatra`,
  previewVideoLink: `somePath`,
  runTime: `1h 30m`,
};

it(`MoviePageDetails is rendered correctly`, () => {
  const tree = renderer.create(<MoviePageOverview
    film={film}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
