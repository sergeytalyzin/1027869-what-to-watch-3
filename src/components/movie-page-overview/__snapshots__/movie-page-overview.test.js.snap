// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageDetails is rendered correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      4.38
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Normal
      </span>
      <span
        className="movie-rating__count"
      >
        865
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    Aliquam erat volutpat.Sed sed nisi sed augue convallis suscipit in sed felis.
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        John Mason
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        John Mason
        Anthony Mann
        Richard Weil
      </strong>
    </p>
  </div>,
]
`;
